src/blocks/blocks/accordion/group/edit.js

```diff
@@ -45,26 +59,125 @@ const Edit = ({
 	}, [ attributes.id ]);
 
+	// Make it true for old users if they have more than one accordion item
+	// with initiallyOpen === true, and false by default otherwise
+	if ( attributes.alwaysOpen === undefined ) {
+		const children = select( 'core/block-editor' ).getBlocksByClientId( clientId )[0].innerBlocks;
+		setAttributes({ alwaysOpen: 1 < children.filter( child => true === child.attributes.initialOpen ).length });
+	}
+
 	const getValue = field => getDefaultValueByField({ name, field, defaultAttributes, attributes });
 
+	const getBorderValue = ( type, position, property ) => {
+		const border = 'header' === type ?
+			getValue( 'headerBorder' ) :
+			( 'content' === type ?
+				getValue( 'contentBorder' ) :
+				undefined );
+
+		if ( ! border ) {
+			return '';
+		}
+
+		// for backward compatibility purposes
+		if ( 'color' === property ) {
+			return border[position] ?
+				border[position].color :
+				border.color || getValue( 'borderColor' );
+		}
+
+		return border[position] && border[position][property] ? border[position][property] : border[property];
+	};
+
+	const boxShadow = getValue( 'boxShadow' );
+
 	const inlineStyles = {
 		'--titleColor': getValue( 'titleColor' ),
 		'--titleBackground': getValue( 'titleBackground' ),
-		'--borderColor': getValue( 'borderColor' ),
-		'--contentBackground': getValue( 'contentBackground' )
+		'--contentBackground': getValue( 'contentBackground' ),
+		'--fontFamily': getValue( 'fontFamily' ),
+		'--fontVariant': getValue( 'fontVariant' ),
+		'--fontStyle': getValue( 'fontStyle' ),
+		'--textTransform': getValue( 'textTransform' ),
+		'--letterSpacing': getValue( 'letterSpacing' ) ? getValue( 'letterSpacing' ) + 'px' : undefined,
+		'--fontSize': getValue( 'fontSize' ) ? getValue( 'fontSize' ) + 'px' : undefined,
+		'--boxShadow': boxShadow.active && `${boxShadow.horizontal}px ${boxShadow.vertical}px ${boxShadow.blur}px ${boxShadow.spread}px ${hex2rgba( boxShadow.color, boxShadow.colorOpacity )}`
 	};
 
+	const addBorderStyle = ( type ) => {
+		[ 'Top', 'Right', 'Bottom', 'Left' ].forEach( position => {
+			[ 'Width', 'Style', 'Color' ].forEach( prop => {
+				if ( 'content' === type && 'Top' === position ) {
+					return;
+				}
+
+				const varName = `--${type.slice( 0, 1 )}Border${position.slice( 0, 1 )}${prop.slice( 0, 1 )}`;
+				inlineStyles[varName] = getBorderValue( type, position.toLowerCase(), prop.toLowerCase() ) || '';
+			});
+		});
+	};
+
+	const addPaddingStyle = ( type ) => {
+		const padding = 'header' === type ? attributes.headerPadding : ( 'content' === type ? attributes.contentPadding : null );
+		if ( ! padding ) {
+			return;
+		}
+
+		[ 'Top', 'Right', 'Bottom', 'Left' ].forEach( position => {
+			const varName = `--${type.slice( 0, 1 )}Padding${position}`;
+			inlineStyles[varName] = padding[ position.toLowerCase() ];
+		});
+	};
+
+	addBorderStyle( 'header' );
+	addBorderStyle( 'content' );
+	addPaddingStyle( 'header' );
+	addPaddingStyle( 'content' );
```

Comment:
This an unsual usage of functions. I recommmend to move this directly into the `inlineStyles`.

And I think you have overenginered this part:

```
border-top: var( --hBorderTW, 1px) var( --hBorderTS, solid) var( --hBorderTC, #d5dadf );
			border-right: var( --hBorderRW, 1px) var( --hBorderRS, solid) var( --hBorderRC, #d5dadf );
			border-bottom: var( --hBorderBW, 1px) var( --hBorderBS, solid) var( --hBorderBC, #d5dadf );
			border-left: var( --hBorderLW, 1px) var( --hBorderLS, solid) var( --hBorderLC, #d5dadf );
```

It can be done with only one CSS var. See:
https://github.com/Codeinwp/otter-blocks/blob/fe63f76c7ef99e8e1f339607701d8efb4206e8af/inc/css/blocks/class-form-css.php#L57-L69

And have a function like this: https://github.com/Codeinwp/otter-blocks/blob/fe63f76c7ef99e8e1f339607701d8efb4206e8af/src/blocks/blocks/form/edit.js#L735 to make the border.